src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,96 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import { SortType } from '../../types/SortType';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todos: Todo[],
+  filteredTodos: Todo[],
+  sortType: SortType,
+  onSelect: (typeOfSort: SortType) => void,
+}
+
+export const TodoFooter: React.FC<Props> = (props) => {
+  const {
+    todos,
+    filteredTodos,
+    sortType,
+    onSelect,
+  } = props;
+
+  const handlerSortSelect = (event: React.MouseEvent<HTMLAnchorElement>) => {
+    const target = event.target as HTMLAnchorElement;
+    const typeOfSort = target.getAttribute('data-sort') as SortType;
+
+    if (typeOfSort === sortType) {
+      return;
+    }
+
+    onSelect(typeOfSort);
+  };
+
+  const hasTodos = filteredTodos.length > 0;
```

Comment:
```suggestion
```
You don't need it here. You can check size of 'todos'. If you really need it, move it to the component above